### PR TITLE
H/display gui

### PR DESCRIPTION
**Pull request by hansely**
gui checkbox

### CODE REVIEW

File: mivisionx_validation_tool.py [hunk 1]
```diff
@@ -603,7 +611,7 @@ def processClassificationOutput(inputImage, modelName, modelOutput, batchSize):
 				print("ERROR: Converting NNIR to OpenVX Failed")
 				quit()
 
-	os.system('(cd '+modelBuildDir+'; cmake ../openvx-files; make; ./anntest ../openvx-files/weights.bin )')
+	#os.system('(cd '+modelBuildDir+'; cmake ../openvx-files; make; ./anntest ../openvx-files/weights.bin )')
```
Comment (LakshmiKumar23) [inline mivisionx_validation_tool.py]: @hansely Can you uncomment this line? It fails to run otherwise

**Comment by LakshmiKumar23** [comment]
@hansely It fails when I click on any particular augmentation to see the stats

File: mivisionx_validation_tool.py [hunk 2]
```diff
@@ -793,86 +812,104 @@ def processClassificationOutput(inputImage, modelName, modelOutput, batchSize):
...
-				print '%30s' % 'Progress image created in ', str((end - start)*1000), 'ms'
-
-			# Plot Graph
+    		#show augmented images
 			start = time.time()
-			accuracy = (float)(correctTop5) / (modelBatchSizeInt*x+i+1) * 100
-			viewer.plotGraph(accuracy)
+			aug_width = final_image_batch.shape[1]
+			aug_height = final_image_batch.shape[0]
+			viewer.showAugImage(final_image_batch, aug_width, aug_height)
```
Comment (LakshmiKumar23) [inline mivisionx_validation_tool.py:859]: @hansely this also needs to go into 'if guiFlag'

File: mivisionx_validation_tool.py [hunk 3]
```diff
@@ -793,86 +812,104 @@ def processClassificationOutput(inputImage, modelName, modelOutput, batchSize):
...
+			else:
+				viewer.setAugName(raliList[progressIndex-1])
+				# Total progress
+				viewer.setTotalProgress(x)
+				# Top 1 progress
+				viewer.setTop1Progress(resultPerAugmentation[progressIndex-1][0], x)
+				# Top 5 progress
+				viewer.setTop5Progress(resultPerAugmentation[progressIndex-1][1], x)
+				# Mismatch progress
+				viewer.setMisProgress(resultPerAugmentation[progressIndex-1][2], x)
```
Comment (LakshmiKumar23) [inline mivisionx_validation_tool.py:891]: 866-899 into 'if guiFlag'

File: mivisionx_validation_tool.py [hunk 4]
```diff
@@ -793,86 +812,104 @@ def processClassificationOutput(inputImage, modelName, modelOutput, batchSize):
...
-		while viewer.isPaused():
-			cv2.waitKey(0)
+			key = cv2.waitKey(2)
 			if not viewer.getState():
+				viewer.stopView()
 				break
+			while viewer.isPaused():
+				cv2.waitKey(0)
+				if not viewer.getState():
+					break
```
Comment (LakshmiKumar23) [inline mivisionx_validation_tool.py:914]: inside 'if guiFlag'

**Review by kiritigowda — APPROVED** [review]

**Pull request merged**